sunpy/net/helioviewer.py

```diff
@@ -390,3 +450,11 @@ def _request(self, params):
     def _format_date(self, date):
         """Formats a date for Helioviewer API requests"""
         return parse_time(date).isot + "Z"
+
+    def _get_header_information(self, URL):
+        response = requests.get(URL)
+        with open('header.xml', 'wb') as file:
+            file.write(response.content)
+        with open('header.xml') as fd:
+            doc = xmltodict.parse(fd.read())
+        return doc
```

Comment:
Do you need to do `with open` twice?

Reply:
Oh yeah sorry. I did that while testing the code on my machine